lib/utils/commands.js

```diff
@@ -0,0 +1,10 @@
+const arguments = require('./cli-flags');
```

Comment:
This is nice, but instead of mutating, maybe `Object.keys` the name and map/filter the names instead?

Reply:
I couldn't follow how `Object.keys` would be helpful here as `cli-flags.js` contains all commands in an array instead of a map.

Reply:
@evenstensberg please let me know how to do it.

Reply:
Object.keys iteration instead of using array.forEach to create those values in another array. Just create one directly

Reply:
Thanks for guidance, Learnt new tricks and about mutability from this task.
Did that in latest commit.